lib/supported-selector.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+// Create a shortcut for the CSSRule interface
+var CSSRule = window.CSSRule
```

Comment:
Why do we use CSSRule here instead of createElement ? Is there any difference? If not I would stay consistent to other supported-* implementations.